scripts/testlib.sh

```diff
@@ -121,6 +121,10 @@ start-monerod-regtest() {
 		--rpc-bind-ip=127.0.0.1 \
 		--rpc-bind-port=18081
 	sleep 5
+	# Make sure the blockchain has some initial decoy outputs. Arbitrarily sending the
+	# rewards to the Mastering Monero address.
+	local rewardsAddr=4BKjy1uVRTPiz4pHyaXXawb82XpzLiowSDd8rEQJGqvN6AD6kWosLQ6VJXW9sghopxXgQSh1RTd54JdvvCRsXiF41xvfeW5
+	mine-monero "${rewardsAddr}" > /dev/null
```

Comment:
When the first test after running `setup-env.sh` had a monero transfer in it, I was getting errors from the transfer about insufficient outputs.  This seemed to fix it.